crates/websocket/src/messages.rs

```diff
@@ -64,12 +65,13 @@ pub struct SendCommunityRoomMessage<OP: ToString, Response> {
   pub websocket_id: Option<ConnectionId>,
 }
 
+/// Send message to mods of a given community. Leave out community_id param to send to site admins.
 #[derive(Message)]
 #[rtype(result = "()")]
 pub struct SendModRoomMessage<Response> {
   pub op: UserOperation,
   pub response: Response,
-  pub community_id: CommunityId,
+  pub community_id: Option<CommunityId>,
```

Comment:
This is pry fine, but I'll need to change this on the front-end too.

Reply:
I think its much clearer this way, because who would guess that you need to set community_id = 0 to reach admins?